tests/conftest.py

```diff
@@ -86,10 +87,20 @@ def assert_valid_schema():
 
 
 @pytest.fixture
-def iopipe():
+def default_iopipe():
     return IOpipe('test-suite', 'https://metrics-api.iopipe.com', True)
 
 
+@pytest.fixture
+def default_iopipe_override():
+    return IOpipe('test-suite', 'https://metrics-api.iopipe.com', True, plugins=[TracePlugin(auto_measure=False)])
```

Comment:
So when someone wants to leave a plugin in kitchen sink out for whatever reason, they use core? (I think this is true and what we would recommend, but want to be sure)

Reply:
Yes, this is how the Node agent works, too. Although, unlike the Node agent, you can override the plugins if you want to change how a preloaded plugin is configured.